### PR TITLE
Dashboard type

### PR DESCRIPTION
**Pull request by Merydian**
Created processing algorithms for all endpoints. Edited GUI to match native QGis. Everything still to be improved.

### CODE REVIEW

File: ohsomeTools/__init__.py [hunk 1]
```diff
@@ -29,7 +29,7 @@
 from datetime import datetime
 
 
-# noinspection PyPep8Naming
+# noinspection PyPep8Namingcis
```
Comment (koebi) [inline ohsomeTools/__init__.py:32]: What does the `cis` part do?

File: ohsomeTools/common/client.py [hunk 2]
```diff
@@ -141,11 +137,18 @@ def request(
...
+
         logger.log(
             "url: {}\nParameters: {}".format(
                 self.url,
                 # final_requests_kwargs
-                json.dumps(body, indent=2),
+                json.dumps(b, indent=2),
             ),
             0,
         )
```
Comment (koebi) [inline ohsomeTools/common/client.py:154]: Does it make sense to log the request here?
Couldn't that be done in the nam.request-method, independent of the invocation?

File: ohsomeTools/gui/ohsome_spec.py [hunk 3]
```diff
@@ -243,60 +223,24 @@ def __prepare_ohsome_time_parameter(
...
-                    str("error"),
-                    # error,
-                    "Found too many input layers with the same name. Use unique names for your layers.",
-                )
-            layer_list.extend(layers)
-        geojsons = [
-            QgsJsonExporter(lyr)
-            .exportFeatures(lyr.getFeatures())
+    def _get_selected_polygon_layers_geometries(self) -> []:
+        layer = self.dlg.layer_input.currentLayer()
```
Comment (koebi) [inline ohsomeTools/gui/ohsome_spec.py:227]: Why are we switching from a list of layers to a single layer here?

**Review by koebi — APPROVED** [review]
Looks good so far - just a few small questions :)
I'd suggest we merge, release and check how/what we want to integration-test.

File: ohsomeTools/__init__.py (hunk 1 above)
Comment (Merydian) [reply ohsomeTools/__init__.py:32]: I'm not sure. I don't remember putting this here, and it's not even in my local code.

File: ohsomeTools/gui/ohsome_spec.py (hunk 3 above)
Comment (Merydian) [reply ohsomeTools/gui/ohsome_spec.py:227]: As discussed in #60 

> The option to add multiple point or polygon layers seems a bit unnecessary and can be dropped

So we no longer need a list to handle multiple layers.

File: ohsomeTools/common/client.py (hunk 2 above)
Comment (Merydian) [reply ohsomeTools/common/client.py:154]: Good point. I'll change it.

File: ohsomeTools/__init__.py (hunk 1 above)
Comment (koebi) [reply ohsomeTools/__init__.py:32]: We'll leave it as is for now.

**Pull request merged**